inputs/make_metadata.R

```diff
@@ -0,0 +1,25 @@
+library(readr)
+library(dplyr)
+library(janitor)
+
+setwd("~/github/2022-txome-minus-gnome/")
+
+doryteuthis <- read_csv("inputs/make_metadata/Doryteuthis-pealeii-sra-run-info.csv") %>%
```

Comment:
ah ok this part is pretty specific and might not be able to be flexible like I suggested before

Reply:
ya there is a way to do what you suggested, but it gets overly complicated and hard to read the snakefile, so just for this first demo repo I hard coded it to keep it as simple as possible.